src/test/java/core/basesyntax/service/ShopServiceTest.java

```diff
@@ -0,0 +1,209 @@
+package core.basesyntax.service;
+
+import static org.hamcrest.CoreMatchers.is;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.activity.ActivityHandler;
+import core.basesyntax.service.activity.BalanceActivityHandler;
+import core.basesyntax.service.activity.PurchaseActivityHandler;
+import core.basesyntax.service.activity.ReturnActivityHandler;
+import core.basesyntax.service.activity.SupplyActivityHandler;
+import core.basesyntax.service.impl.ActivityStrategyImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ShopServiceTest {
```

Comment:
It's difficult to read and understand your tests with variables like this. Can you use local variables instead?